.hadolint.yaml

```diff
@@ -17,6 +17,9 @@
 #and add in this format: - DL3000
 ignored:
 - DL3006
+- DL3018
+- DL3008
+
```

Comment:
I don't think we need to have an extra line here.

Reply:
removed extra line